lib/audits-analyzer.js

```diff
@@ -0,0 +1,71 @@
+/**
+ *  Copyright (c) 2018 Alex Smagin <me@asmagin.com> (https://github.com/asmagin)
+ *
+ * This software is released under the MIT License.
+ * https://opensource.org/licenses/MIT
+ */
+
+const chalk = require('chalk');
+
+const analyzeAudit = (audit, threshold) => {
+  switch (audit.scoreDisplayMode) {
+    case 'numeric':
+      return Number(audit.numericValue) <= Number(threshold);
+    case 'informative':
+      // if cannot be checked (NaN) - skip
+      return (
+        isNaN(Number(audit.numericValue)) ||
+        Number(audit.numericValue) <= Number(threshold)
+      );
+    default:
+      return !!audit.score === !!threshold;
```

Comment:
This will be correct for categories scores, but most of the audit values are times, sizes, amount of requests. That is why I'm applying the lower the better logic.